content/accessibility/guidelines.mdx

```diff
@@ -88,13 +88,13 @@ There should only be one `h1` per web page. (Markdown on a page is an exception)
 
 <DoDontContainer stacked>
     <Do>
-      {![""](https://user-images.githubusercontent.com/40274682/102439796-cb83e480-3fd3-11eb-824a-e2f819bf55ff.png)}
-        <Caption>Follow a clear hearing hierarchy</Caption>
+       <img src="https://user-images.githubusercontent.com/40274682/102439796-cb83e480-3fd3-11eb-824a-e2f819bf55ff.png" alt="" />
+       <Caption>Follow a clear hearing hierarchy</Caption>
     </Do>
     <Dont>
-      {![""](https://user-images.githubusercontent.com/40274682/102439772-be66f580-3fd3-11eb-8ab2-1f6ca42bc745.png)}
-        <Caption>Don't use multiple H1s, skip over heading hierarchy for visual effect, or forget to label sections of information with headings</Caption>
-    </Dont>    
+       <img src="https://user-images.githubusercontent.com/40274682/102439772-be66f580-3fd3-11eb-8ab2-1f6ca42bc745.png" alt="" />
+       <Caption>Don't use multiple H1s, skip over heading hierarchy for visual effect, or forget to label sections of information with headings</Caption>
+    </Dont>
 </DoDontContainer>
```

Comment:
Bat signal to @colebemis...help 😅. This is breaking vercel deployment and I'm not sure where to find the right syntax to properly put images in a <dodontcontainer>

Reply:
It was not this. It was the wonky </br> I had in there 🙈